pints/tests/test_abc_smc.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+#
+# Tests the basic methods of the adaptive covariance MCMC routine.
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+import pints
```

Comment:
Please order these alphabetically.

(Ideally, you'd also go 1. standard library, empty line, 2. used libraries, empty line, 3. local imports (pints))

Reply:
I have opted for going with standard library, empty line, used library, empty line and local imports